docs/iaas.md

```diff
@@ -98,3 +102,68 @@ http://localhost:30000/d/openstack-overview
 - Use the following credentials:
   - username: `admin`
   - password: `pass`
+
+### Ceph
+
+The SCS IaaS reference implementation (OSISM) currently supports two methods for deploying Ceph:
+- [ceph-ansible](https://github.com/ceph/ceph-ansible)
+- [rook operator](https://github.com/rook/rook)
+
+This guide covers Ceph cluster monitoring for both deployment methods. While both expose the same metrics via the same
+endpoint, there are some differences in Prometheus configuration and alerts.
+
+#### Prometheus metrics and alerts
+
+Ceph contains 2 build-in sources of metrics a.k.a. exporters.
+The Ceph exporter (introduced in Reef release of Ceph) is the main source of Ceph performance metrics. It runs as a
```

Comment:
according to the docs this exporter is also available in the quincy ceph release:

https://docs.ceph.com/en/quincy/monitoring/#ceph-metrics

Reply:
according to the release notes the ceph exporter has been introduced in the Reef release https://docs.ceph.com/en/latest/releases/reef/#dashboard

Additionally, I've tested both releases and can confirm that the Quincy Ceph release does not contain that exporter daemon.

Reply:
well, it's in the complete docs, see the URL, which indicates these are the docs for the quincy release, and they contain the ceph-exporter:

> Performance counters are transformed into standard Prometheus metrics by the Ceph exporter daemon. This daemon runs on every Ceph cluster host and exposes a metrics end point where all the performance counters exposed by all the Ceph daemons running in the host are published in the form of Prometheus metrics.

maybe the docs are just buggy, I didn't investigate.